bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/connector/Executor.java

```diff
@@ -185,6 +187,8 @@ public void notifyFailure(ErrorValue error) {
             return double.class;
         } else if (paramValue instanceof Float) {
             return double.class;
+        } else  if (paramValue instanceof ArrayValueImpl) {
```

Comment:
Can you check the formatting here? Seems this is not formatted correctly.

Reply:
Corrected the formatting for the Executor class